app/tests/unit/pages/project-revision/[projectRevision]/index.test.tsx

```diff
@@ -157,10 +157,27 @@ describe("The Create Project page", () => {
 
   it("Renders the summary with the filled-out details", async () => {
     pageTestingHelper.loadQuery({
-      Form() {
-        return {
-          jsonSchema: projectContactProdSchema,
-        };
+      Form(context) {
+        // If the form is the project manager form, return the project manager schema
+        if (
+          context.name === "formByJsonSchemaName" &&
+          context.path[2] == "projectManagerFormChangesByLabel"
```

Comment:
== vs. === is necessary or typo?

Reply:
Not entirely necessary but this way it ensures the typing is correct as well as equality.